airgoons_mikeforce_customizations/config/arsenal.hpp

```diff
@@ -401,8 +400,9 @@ class vn_whitelisted_arsenal_loadouts
 			{"vn_welrod_mag",{-1,0,-1,-1}},
 			{"murshun_cigs_matches",{0,0,0,0}},
 			{"vn_b_item_cigs_01", {0,0,0,0}},
-			{"murshun_cigs_cigpack", {2,2,2,2}},
+			{"murshun_cigs_cigpack", {2,0,2,2}},
 			{"immersion_cigs_cigar0_nv", {-1,-1,3,-1}},
+			{"immersion_cigs_cigar0", {-1,-1,3,-1}},
```

Comment:
Should cigars be {-1,3,3,-1}?

Reply:
Normally yes but restrict by side is turned off in Mike Force so it just uses the lowest in any category.

Reply:
Good to know, thanks